### DISCUSSION TITLE
タイトなリストに表をネストすると、表がリストの文章内に含まれる

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#555] [要望・アイデア・不具合]
タイトなリスト内に表をネストすると、表がリストの文章内に含まれてしまいます。

参考：https://qiita.com/Yarakashi_Kikohshi/private/7ed441495d0d24be6871

まず、リスト文章が短く表が小さい以下の場合を考えます。

```markdown
- item1
    |This is|
    |:-----:|
    |a table|
- item2
```

![image](https://github.com/increments/qiita-discussions/assets/75878492/4156d37a-3a8f-4ec4-af25-d96da748b96f)

一方で、GitHub では以下のような結果を得ます。このような結果を期待しています。

- item1
    |This is|
    |:-----:|
    |a table|
- item2

-----

次のように、文章と表がある程度大きい場合、ウィンドウ幅によって表の場所が変化します。

| OKな例 | NGな例 |
|--------|--------|
| ![ok][] | ![ng][] |

[ng]: https://github.com/increments/qiita-discussions/assets/75878492/2aeb6921-37a3-4713-bb86-54195c3074d7
[ok]: https://github.com/increments/qiita-discussions/assets/75878492/e9bdb077-d9d9-4a40-a3b5-e199d2fef12b

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
リクエストありがとうございます！
こちら調査の上、検討します！
